src/components/GetAppPage/GetAppPage.vue

```diff
@@ -25,13 +19,19 @@
 
     <section class="feature">
       <div>
-        <div v-for="(feature, key) in $t('GetAppPage.Feature')" :key="key" class="container side-by-side">
+        <div v-for="number in valueProps" :key="number" class="container side-by-side">
```

Comment:
Vue can do this~
```suggestion
        <div v-for="number in 3" :key="number" class="container side-by-side">
```

Reply:
wow! so lean

Reply:
https://vuejs.org/v2/guide/list.html#v-for-with-a-Range